configure.py

```diff
@@ -16,14 +16,7 @@
 import sys
 from pathlib import Path
 from typing import Any, Dict, List
-
-from tools.project import (
-    Object,
-    ProjectConfig,
-    calculate_progress,
-    generate_build,
-    is_windows,
-)
+from tools.project import *
```

Comment:
I missed this before, I'm not a big fan of ``from X import *`` from my fast64 experience (which had these everywhere and caused issues)

though I guess it's not that important but I wanted to mention it

Reply:
Given that configure.py/project.py are closely tied, I wasn't too worried about this. I totally get the reasoning though.